gr-audio/lib/windows/windows_sink.cc

```diff
@@ -231,8 +231,8 @@ MMRESULT windows_sink::is_format_supported(LPWAVEFORMATEX pwfx, UINT uDeviceID)
     return (waveOutOpen(NULL,                // ptr can be NULL for query
                         uDeviceID,           // the device identifier
                         pwfx,                // defines requested format
-                        NULL,                // no callback
-                        NULL,                // no instance data
+                        0,                   // no callback
```

Comment:
Can these be `nullptr`?

Reply:
I don't think so, because `DWORD_PTR` is an integer type, not a pointer type.